src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js

```diff
@@ -10,7 +10,8 @@
 
     /* ngInject */
     function OverviewController($scope, $stateParams, $q, $geolocation, leafletData,
-                                Directions, Map, MapControl, MapStyle, NavbarConfig, Notifications) {
+                                Config, Directions, Map, MapControl, MapStyle, NavbarConfig, 
+                                Notifications) {
```

Comment:
Yeah, its getting up there. Unfortunately this is the view that has a bit of everything.